### PR TITLE
Librerialterna

### PR DESCRIPTION
**Pull request by jeffrey20122021**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "mochawesome": "^7.1.3"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "isomorphic-fetch": "^3.0.0"
```
Comment (holgiosalos) [inline package.json:41]: install isomorphic-fetch as devDependency

File: test/schema/GithubApi.Gist.Fetch.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  }
+};
+
+chai.use(chaiSubset);
+
+const fetch = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
```
Comment (holgiosalos) [inline test/schema/GithubApi.Gist.Fetch.test.js]: the idea of this point is to use the isomorphic-fetch library instead of axios.

**Review by holgiosalos — COMMENTED** [review]
Run the lint fix command to fix the issues on the continuos integration pipeline

File: test/schema/GithubApi.Gist.Fetch.test.js (hunk 2 above)
Comment (jeffrey20122021) [reply test/schema/GithubApi.Gist.Fetch.test.js]: done @holgiosalos

File: package.json (hunk 1 above)
Comment (jeffrey20122021) [reply package.json:41]: This was already set up

**Review by holgiosalos — APPROVED** [review]
Congrats!

**Pull request merged**